fenzo-core/src/main/java/com/netflix/fenzo/ActiveVmGroups.java

```diff
@@ -52,13 +52,25 @@ private String getName() {
         activeVmGroupsMap.put(0, new ArrayList<VmGroup>());
     }
 
+    /**
```

Comment:
In general, we skip writing javadoc comments for private methods. Are you adding this for a specific reason? This method seems simple enough to understand without the comments. We could accept this if you have a specific reason to add these comments. Thank you.